docs/api.md

```diff
@@ -482,12 +483,14 @@ class MyAsyncRemoteTransformer extends Kinto.transformers.RemoteTransformer {
   }
 }
 
-coll.use(new MyAsyncRemoteTransformer());
+coll = kinto.collection("articles", {
+  remoteTransformers: [ new MyAsyncRemoteTransformer() ]
+});
 ```
 
 ### Multiple transformers
 
-Transformers are stacked when `#use()` is called multiple times, in the order of the calls; that means you can chain multiple encoding operations, with the decoding ones being processed in the reverse order:
+The remoteTransformers field of the plugins object passed to `Kinto#collection` is an Array. That means you can chain multiple encoding operations, with the decoding ones being processed in the reverse order:
```

Comment:
Maybe we should document it like this:
### Kinto-level options:
* remote (String)
* headers (Object with Strings)
* adapter (plugin)
* requestMode (String?)
* bucket (String)
* dbPrefix (String)

### Collection-level options:
* idSchema (plugin)
* remoteTransformers (Array of plugins)
* localTransformers (Array of plugins)

And then for symmetry, instead of having `Kinto.createRemoteAdapter` and `Kinto.createIdSchema`, we could eventually have `Kinto.createPlugin` which would cover those two and also [custom adapters](http://kintojs.readthedocs.org/en/latest/extending/#custom-database-adapters), as an ES5 alternative to extending `Kinto.adapters.BaseAdapter`, `Kinto.transformers.RemoteTransformer`, and the planned `Kinto.transformers.LocalTransformer` and (I guess) `Kinto.idSchemas.BaseIdSchema`.

So if you agree that this is a sensible way to name things, I'll rephrase:

    The remoteTransformers field of the plugins object passed to `Kinto#collection`

as:

    The remoteTransformers field of the collection-level options object